crates/sui-framework/packages/sui-framework/sources/coin.move

```diff
@@ -260,6 +275,40 @@ module sui::coin {
         )
     }
 
+    /// This creates a new currency, via `create_currency`, but with an extra capability that
+    /// allows for specific addresses to have their coins frozen. Those addresses will not
+    /// be able to interact with the coin
```

Comment:
Do we want to go into more detail on what "interact" means?

Reply:
I think we should, though I'm not entirely sure what the limitations are

Reply:
@lxfind, we should fill this in as your work proceeds